### ISSUE TITLE
Frontline transport

### ISSUE THREAD

**Issue by SomethingSimpl** [shdwp/dcs_liberation#25]
Hi,

The frontline transport mission doesnt seem to be working for us. We land by the troops but noone gets into the helicopter so we cant transport them.

Is there a specific thing we need to do? Or is it just currently bugged?

**Comment by shdwp** [comment]
You have to use the comms menu to get the infantry going. I don't remember exact items naming, but shouldn't be that hard to find.

**Comment by zbenesch** [comment]
Same here, no comm items, no ctld script, no triggers in the mission.
I was wondering the same thing, what do I need to do to get them embarked.

**Comment by SomethingSimpl** [comment]
Can we maybe get the issue reopened?

**Comment by shdwp** [comment]
K, I'll double check that.

**Comment by zbenesch** [comment]
Any word on this?

**Comment by shdwp** [comment]
No, not yet. I plan on redoing and greatly improving heli part in next major release, so this one is postponed until that. For now you can check with Easy comms.
